plugins/kubernetes-react/src/components/Pods/PodDrawer/ContainerCard.tsx

```diff
@@ -41,6 +41,16 @@ const getContainerHealthChecks = (
       'no restarts': containerStatus.restartCount === 0,
     };
   }
+  if (containerSpec && containerSpec?.livenessProbe !== undefined) {
```

Comment:
Can we avoid the duplication of the conditions? for example, define the checks in the return object, if the liveness probe is defined add the new check to the object, return the object